pysodium/__init__.py

```diff
@@ -409,6 +417,15 @@ def crypto_sign_seed_keypair(seed):
     __check(sodium.crypto_sign_seed_keypair(pk, sk, seed))
     return pk.raw, sk.raw
 
+def crypto_sign_ed25519_sk_to_seed(sk):
+    seed = ctypes.create_string_buffer(crypto_sign_SEEDBYTES)
+    __check(sodium.crypto_sign_ed25519_sk_to_seed(seed, sk))
+    return seed.raw
+
+def crypto_sign_ed25519_sk_to_pk(sk):
```

Comment:
how is this different from crypto_sign_sk_to_pk in line 522 of the unpatched file?